python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -908,11 +912,21 @@ def run_cutouts(args):
         sources = cutouts.run(data, butler, njobs=args.jobs)
     else:
         sources = []
-        count = len_sources(apdb_query)
+        all_data = []
+        cols_to_export = ["diaSourceId", "ra", "dec", "midpointMjdTai"]
+        count = len_sources(apdb_query, args.namespace)
         for i, data in enumerate(getter):
+            all_data.append(data[cols_to_export])
             sources.extend(cutouts.write_images(data, butler, njobs=args.jobs))
             print(f"Completed {i+1} batches of {args.limit} size, out of {count} diaSources.")
         cutouts.write_manifest(sources)
+        # Export diaSource dataframe to a CSV if `save_as_numpy` is enabled.
+        if config.save_as_numpy:
+            dia_sources_df = pd.concat(all_data)
+            # Filter out any errored diaSourceIds.
+            dia_sources_df = dia_sources_df[dia_sources_df['diaSourceId'].isin(sources)]
+            dia_sources_df[cols_to_export].to_csv(
```

Comment:
```suggestion
            dia_sources_df.to_csv(
```
by construction you only have `cols_to_export` already

Reply:
Thanks for pointing it out. That was a silly one. Fixed!